src/App.vue

```diff
@@ -32,11 +42,23 @@ export default defineComponent({
         store.dispatch("UserState/checkUserState", true)
       );
     });
+
+    watch(
```

Comment:
i think, will be better if you add additional meta to route, something like "layout". And the layout dependence from the meta. And will be cool, if 'layout component' will connect dynamically